### ISSUE TITLE
roi contains non-integers crash

### ISSUE THREAD

**Issue by akreshuk** [ilastik/volumina#174]
Pixel classification with 3D data, input data transposed from zyx to xyz. Labeling works for some time and then suddenly starts complaining and doesn't work anymore:

```pytb
ERROR 2016-12-05 16:45:39,289 excepthooks 10862 140099317155648 Unhandled exception in thread: 'MainThread'
ERROR 2016-12-05 16:45:39,290 excepthooks 10862 140099317155648 Traceback (most recent call last):
  File "/home/akreshuk/miniconda2/envs/ilastik-devel-new/ilastik-meta/volumina/volumina/brushingcontroller.py", line 292, in _writeIntoSink
    self._dataSink.put(slicing, labels.reshape(tuple(newshape)))
  File "/home/akreshuk/miniconda2/envs/ilastik-devel-new/ilastik-meta/volumina/volumina/pixelpipeline/datasources.py", line 361, in put
    self._inputSlot[transposedSlicing] = transposedArray.view(np.ndarray)
  File "/home/akreshuk/miniconda2/envs/ilastik-devel-new/ilastik-meta/lazyflow/lazyflow/slot.py", line 1018, in __setitem__
    roi = self.rtype(self, pslice=key)
  File "/home/akreshuk/miniconda2/envs/ilastik-devel-new/ilastik-meta/lazyflow/lazyflow/rtype.py", line 132, in __init__
    assert isinstance(start, (int, long, numpy.integer)), "Roi contains non-integers: {}".format( self )
AssertionError: Roi contains non-integers: Subregion: start '[383, 266, 20.0, 0]' stop '[593, 304, 21.0, 1]'
```
I suspect it has something to do with Python2->3 changes?

**Comment by stuarteberg** [comment]
>I suspect it has something to do with Python2->3 changes?

Ah, good theory.  I guess we should look for files that contain `from future import __division__`, and then find the offending line.

**Comment by akreshuk** [comment]
Changing line 280 of brushingcontroller.py to:
`slicing.insert(activeView, slice(int(slicingPos[activeView]), int(slicingPos[activeView]+1)))`
makes it go away for me, but it was not always triggered anyway, so it might be just luck. I'll try to find the root issue.

**Comment by akreshuk** [comment]
Seems to be fixed by ab4bc587c849eabb1fd6622561768adaf3e7f5fb, at least I can't get it to trigger any more. I left the brushingcontroller fix from above in as well, for safety.
